data/cpu_bugs/vm_install_script/sle-15/create_vm_url.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ $# -lt 4 ]; then
+    echo "$0 <name> <install_url> <autoyast_url> <logfile_path> <vm_pool> [cpu]"
+    exit 1
+fi
+
+NAME=$1
+QCOW2POOL=$5
+LOGFILE=$4
+INSTALL_URL=$2
+AUTOYAST_URL=$3
+CPU=$6
+if [ -z $CPU ]; then
+    CPU=host-model-only
+fi
```

Comment:
Might be easier to use required arguments and default arguments though, e.g.

```
name="${1:?"name parameter required"}"
…
cpu="${6:-"host-model-only"}"
```

Reply:
Got it. I will use ShellCheck, and also improve it.